src/jupyter_contrib_nbextensions/nbextensions/code_font_size/code_font_size.yaml

```diff
@@ -1,5 +1,6 @@
 Type: IPython Notebook Extension
 Name: Code Font Size
+Link: readme.md
```

Comment:
this may need to be uppercase, to match the file name. I'm not sure how the notebook handles case sensitivity in urls...